src/user/user.controller.ts

```diff
@@ -49,6 +51,7 @@ export class UserController {
   @ApiNotFoundResponse({ description: "The record was not found." })
   @ApiForbiddenResponse({ description: "Forbidden!" })
   @UseRoles(Role.ADMIN)
+  @UseInterceptors(ClassSerializerInterceptor)
```

Comment:
Same as in the `UserEntity` remove all attempts to serialize the responses, including imports as we have a separate issue for it

Reply:
Will do